Dockerfile

```diff
@@ -20,7 +20,7 @@ USER software
 
 COPY --chown=software:software . .
 
-RUN corepack install --global pnpm
+RUN corepack install --global pnpm@9.15.3
```

Comment:
The pnpm version is automatically updated from renovatebot. So I don't think this is very maintainable. I think this in the base image:

```Dockerfile
ENV COREPACK_DEFAULT_TO_LATEST=0
RUN corepack install --global pnpm
```

in the base, and 

```Dockerfile
ENV COREPACK_ENABLE_NETWORK=0
```
In the production should also do the trick. But I'll test this